conf/evolutions/126-job-resumed-by-super-user.sql

```diff
@@ -0,0 +1,12 @@
+START TRANSACTION;
+
+do $$ begin ASSERT (select schemaVersion from webknossos.releaseInformation) = 125, 'Previous schema version mismatch'; end; $$ LANGUAGE plpgsql;
+
+DROP VIEW webknossos.jobs_;
+
+ALTER TABLE webknossos.jobs ADD column resumedBySuperUser boolean NOT NULL DEFAULT false;
```

Comment:
I know, very nitpicky but `boolean` should be in caps here as well, correct?

```suggestion
ALTER TABLE webknossos.jobs ADD column resumedBySuperUser BOOLEAN NOT NULL DEFAULT false;
```